### PR TITLE
Add instructions for installing the GraalVM through sdkman.

### PR DESCRIPTION
**Pull request by dandreadis**
I think there are many people using sdkman to manage their java installations and that was the steps I've had to use for my local install.

### CODE REVIEW

File: _guides/building-native-image-guide.adoc [hunk 1]
```diff
@@ -25,13 +25,20 @@ Version {graalvm-version} is required.
...
+Once you have downloaded GraalVM, expand the archive and set the `GRAALVM_HOME` variable to its location:
 
 `export GRAALVM_HOME=$HOME/Development/graalvm/`
 
 On MacOS, point the variable to the `Home` sub-directory:
 
 `export GRAALVM_HOME=$HOME/Development/graalvm/Contents/Home/`
+
+You can also install GraalVM using https://sdkman.io/[sdkman].
+For example, to install the 1.0.0-RC12 version of GraalVM use:
```
Comment (cescoffier) [inline _guides/building-native-image-guide.adoc]: Can you use the variable configuring the GraalVM version

File: _guides/building-native-image-guide.adoc [hunk 2]
```diff
@@ -25,13 +25,20 @@ Version {graalvm-version} is required.
...
+
+`sdk install java 1.0.0-rc-12-grl`. Then set the `GRAALVM_HOME` variable accordingly:
```
Comment (cescoffier) [inline _guides/building-native-image-guide.adoc]: And here is the trick... They use a different format for the version. So we need a second variable.

**Comment by cescoffier** [comment]
BTW this PR should be open against the main Quarkus repository (I know it's confusing). But I can apply it there.

**Comment by emmanuelbernard** [comment]
Right I'll close this one as these changes must go in the main quarkus repo @dandreadis

**Pull request closed without merging**